pkg/kubelet/gpu/nvidia/nvidia_gpu_manager.go

```diff
@@ -239,7 +239,7 @@ func (ngm *nvidiaGPUManager) gpusInUse() *podGPUs {
 		var containersToInspect []containerIdentifier
 		for _, container := range pod.Status.ContainerStatuses {
 			if containers.Has(container.Name) {
-				containersToInspect = append(containersToInspect, containerIdentifier{container.ContainerID, container.Name})
+				containersToInspect = append(containersToInspect, containerIdentifier{strings.Replace(container.ContainerID, "docker://", "", 1), container.Name})
```

Comment:
I'm very surprised that this would not happen by default. @yujuhong Is this a subtle bug from dockertools -> dockershim ?